pywasm/execution.py

```diff
@@ -1946,6 +1771,20 @@ def f64_reinterpret_i64(config: Configuration, i: binary.Instruction):
         config.stack.append(a)
 
 
+def _make_instruction_table():
```

Comment:
I think this table should be written by hand instead of using `dir`, `getattr` or other tricks: this will make the code unmaintainable.

Reply:
I think this form is more maintainable because it conforms to the DRY principle, maybe there need some comments.
but I decide to respect your opinion, now it is written by hand.